Gemfile

```diff
@@ -24,6 +24,11 @@ gem 'publify_amazon_sidebar', path: 'publify_amazon_sidebar'
 gem 'publify_core', path: 'publify_core'
 gem 'publify_textfilter_code', path: 'publify_textfilter_code'
 
+gem 'tzinfo-data'
```

Comment:
What's this for?

Reply:
I can't build bundle without it. Maybe a runtime dependency? From sqlite?

Reply:
```
TZInfo::DataSourceNotFound: tzinfo-data is not present. Please add gem 'tzinfo-data' to your Gemfile and run bundle install
/publify/config/environment.rb:5:in `<top (required)>'
/root/.rbenv/versions/2.4.2/bin/bundle:23:in `load'
/root/.rbenv/versions/2.4.2/bin/bundle:23:in `<main>'
TZInfo::ZoneinfoDirectoryNotFound: None of the paths included in TZInfo::ZoneinfoDataSource.search_path are valid zoneinfo directories.
/publify/config/environment.rb:5:in `<top (required)>'
```
So… I'll let it here.